Cargo.toml

```diff
@@ -33,11 +36,19 @@ version = "1"
 package = "embedded-hal"
 version = "0.2"
 
+[dependencies.eh1]
+package = "embedded-hal"
+version = "1.0.0-rc.3"
+optional = true
```

Comment:
It's OK with me to make this a non-optional dependency. What's the benefit of keeping it optional?

No matter its mandatory / optional status, we should include some `lib.rs` top-level docs about its support. And if it's optional, users might want to know the `eh1` feature name so they can turn it on.

Reply:
If it isn't optional, an `rc.4` might wasm or create without even a code change by us. We have experienced that in RTIC before; I would remove the feature flag once eh1 is properly released. I also wouldn't document it until then. (will probably happen within a couple of weeks)